_security-plugin/authentication-backends/authc-index.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: default
+title: Authentication backends
+nav_order: 45
+has_children: true
+has_toc: false
+redirect_from:
+  - /security-plugin/authentication-backends/
+---
+
+# Authentication backends
+
+Authentication backend configurations determine the method or methods you use for authenticating users and the way users pass their credentials and sign in to OpenSearch. Having an understanding of the basic authentication flow before getting started can help with the configuration process for whichever backend you choose. Consider the high-level sequence of events in the description that follows, and then refer to the detailed steps for configuring the authentication type you choose to use with OpenSearch.
```

Comment:
Thoughts on: `.. the method/s to be used for authenticating users ..`

Reply:
Yes, singular usage sounds good and it won't divert the user interpretation of this sentence.